config.toml

```diff
@@ -8,4 +8,6 @@ ogmios.address = "127.0.0.1"
 ogmios.port = 1337
 
 firstFetchBlock.slot = 44366242
-firstFetchBlock.id = "d2a4249fe3d0607535daa26caf12a38da2233586bc51e79ed0b3a36170471bf5"
\ No newline at end of file
+firstFetchBlock.id = "d2a4249fe3d0607535daa26caf12a38da2233586bc51e79ed0b3a36170471bf5"
+
+# datumFilterPath = "filter.json"
```

Comment:
Is this included as a comment to show how to use it in `config.toml`? If so, could you add an explanatory comment and possibly reiterate that no filter path implies saving all datums?

Reply:
It's in readme, my next PR will change it a bit and more detailed explanation will be there